docs/assemblies/assembly-record-encryption-filter.adoc

```diff
@@ -0,0 +1,29 @@
+// file included in the following:
+//
+// index.adoc
+
+[id='assembly-record-encryption-filter-{context}']
+= Record Encryption filter
+
+[role="_abstract"]
+Streams for Apache Kafka Proxy's Record Encryption filter enhances the security of Kafka messages.
```

Comment:
```suggestion
Kroxylicious's Record Encryption filter enhances the security of Kafka messages.
```

Reply:
Should we be extracting a variable so that `Kroxylicious` can be replaced downstream?